src/main/java/com/rpg2014/wrappers/MinecraftDynamoWrapper.java

```diff
@@ -0,0 +1,162 @@
+            key = new HashMap<>();
+            return getItem(key, SERVER_RUNNING).get(VALUE).bool();
+        } catch (DynamoDbException e) {
+            throw new InternalServerErrorException("Failed to get isServerRunning");
+        }
+    }
+
+    public void setServerRunning() {
+        try {
+            key = new HashMap<>();
+            updatedValues = new HashMap<>();
+            setItem(key, updatedValues, SERVER_RUNNING, true);
+        } catch (DynamoDbException e) {
+            throw new InternalServerErrorException("Failed to setServerRunning");
+        }
+    }
+
+    public void setServerStopped() {
+        try {
+            key = new HashMap<>();
+            updatedValues = new HashMap<>();
+            setItem(key, updatedValues, SERVER_RUNNING, false);
+        } catch (DynamoDbException e) {
+            throw new InternalServerErrorException("Failed to setServerStopped");
+        }
+    }
+
+    public String getSnapshotId() {
+        try {
+            key = new HashMap<>();
+            return getItem(key, SNAPSHOT_ID).get(VALUE).toString();
+        } catch (DynamoDbException e) {
+            throw new InternalServerErrorException("Failed to getSnapshotId");
+        }
+    }
+
+    public void setSnapshotId(final String snapshotId) {
+        try {
+            key = new HashMap<>();
+            updatedValues = new HashMap<>();
+            setItem(key, updatedValues, SNAPSHOT_ID, snapshotId);
+        } catch (DynamoDbException e) {
+            throw new InternalServerErrorException("Failed to setSnapshotId");
+        }
+    }
+
+    public String getInstanceId() {
+        try {
+            key = new HashMap<>();
+            return getItem(key, INSTANCE_ID).get(VALUE).toString();
+        } catch (DynamoDbException e) {
+            throw new InternalServerErrorException("Failed to getInstanceId");
+        }
+    }
+
+
+    public void setInstanceId(final String instanceId) {
+        try {
+            key = new HashMap<>();
+            updatedValues = new HashMap<>();
+            setItem(key, updatedValues, INSTANCE_ID, instanceId);
+        } catch (DynamoDbException e) {
+            throw new InternalServerErrorException("Failed to setInstanceId");
+        }
+    }
+
+    public String getAmiID() {
+        try {
+            key = new HashMap<>();
+            return getItem(key, AMI_ID).get(VALUE).toString();
+        } catch (DynamoDbException e) {
+            throw new InternalServerErrorException("Failed to getAmiId");
+        }
+    }
+
+    public void setAmiId(final String amiId) {
+        try {
+            key = new HashMap<>();
+            updatedValues = new HashMap<>();
+            setItem(key, updatedValues, AMI_ID, amiId);
+        } catch (DynamoDbException e) {
+            throw new InternalServerErrorException("Failed to setAmiId");
+        }
+    }
+
+    private Map<String, AttributeValue> getItem(HashMap<String, AttributeValue> map, final String str) {
```

Comment:
You always pass in a empty map for the first parameter.  Remove it as a paramenter and just make an new map inside the funciton

Reply:
Also rename str to itemId or key or something